src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -2501,32 +2530,40 @@ public long getLastKeepAlive() {
     }
 
     /**
-     * Gets the packet to add the player from the tab-list.
+     * Sends the packet to render all the players that are online.
      *
-     * @return a {@link PlayerInfoPacket} to add the player
      */
-    @NotNull
-    protected PlayerInfoPacket getAddPlayerToList() {
+    protected void getAllPlayerToList() {
```

Comment:
`get` feels a bit misleading as it doesn't return anything

Reply:
Rename to something like `sendAllPlayerInList()`